src/domain/community/user/user.service.ts

```diff
@@ -545,12 +542,17 @@ export class UserService {
 
       user.communicationID = communicationID;
 
-      await this.userRepository.save(user);
+      await this.save(user);
       await this.setUserCache(user);
     }
 
     return user;
   }
+
+  async save(user: IUser): Promise<IUser> {
+    return await this.userRepository.save(user);
```

Comment:
User implements IUser, but is not the same. If a DTO has more properties from `User` than the interface , these properties won't be updated in the database.

Reply:
I think you are correct, but this is the same pattern that we have on all entities - so we do want to fix it then we need to do it structurally - not just here. So would suggest to take it separately from this PR.